sway/commands/unmark.c

```diff
@@ -0,0 +1,32 @@
+#define _POSIX_C_SOURCE 200809L
+#include <string.h>
+#include "sway/commands.h"
+#include "sway/config.h"
+#include "sway/tree/view.h"
+#include "list.h"
+#include "log.h"
+#include "stringop.h"
+
+struct cmd_results *cmd_unmark(int argc, char **argv) {
+	if (argc == 0) {
+		// Remove all marks from the current container
```

Comment:
Sorry. I was incorrect in my statement previously. `unmark` should remove all marks from all views unless criteria is given. Example: `[con_id=__focused__] unmark` would only remove marks from the focused view